plugins/lime-plugin-fbw/src/style.less

```diff
@@ -1,3 +1,47 @@
 button:disabled {
     opacity: 0.3;
 }
+
```

Comment:
For the future, we need to define a Design System for the LimeApp, in which general rules for type and spacing are defined, so we avoid each component having specific rules.
To show you what I mean: https://tailwindcss.com/docs/utility-first